elliott/elliottlib/gitdata.py

```diff
@@ -1,262 +0,0 @@
-                                   'You must either clear your local data or manually checkout the correct branch.'
-                                   ).format(branch, self.branch)
-                            raise GitDataBranchException(msg)
-                    else:
-                        # Check if local is synced with remote
-                        rc, out, err = exectools.cmd_gather(["git", "ls-remote", self.data_path, self.branch])
-                        if rc:
-                            raise GitDataException('Unable to check remote sha: {}'.format(err))
-                        remote = out.strip().split('\t')[0]
-                        try:
-                            exectools.cmd_assert('git branch --contains {}'.format(remote))
-                            self.logger.info('{} is already cloned and latest'.format(self.data_path))
-                            clone_data = False
-                        except:
-                            if not synced:
-                                msg = ('Local data is out of sync with remote and you have unpushed commits: {}\n'
-                                       'You must either clear your local data\n'
-                                       'or manually rebase from latest remote to continue'
-                                       ).format(data_destination)
-                                raise GitDataException(msg)
-
-            if clone_data:
-                if os.path.isdir(data_destination):  # delete if already there
-                    shutil.rmtree(data_destination)
-                self.logger.info('Cloning config data from {}'.format(self.data_path))
-                if not os.path.isdir(data_destination):
-                    cmd = "git clone -b {} --depth 1 {} {}".format(self.branch, self.data_path, data_destination)
-                    rc, out, err = exectools.cmd_gather(cmd)
-                    if rc:
-                        raise GitDataException('Error while cloning data: {}'.format(err))
-
-            self.remote_path = self.data_path
-            self.data_path = data_destination
-        elif data_url.scheme in ['', 'file']:
-            self.remote_path = None
-            self.data_path = os.path.abspath(self.data_path)  # just in case relative path was given
-        else:
-            raise ValueError(
-                'Invalid data_path: {} - invalid scheme: {}'
-                .format(self.data_path, data_url.scheme)
-            )
-
-        if self.sub_dir:
-            self.data_dir = os.path.join(self.data_path, self.sub_dir)
-        else:
-            self.data_dir = self.data_path
-        if not os.path.isdir(self.data_dir):
-            raise GitDataPathException('{} is not a valid sub-directory in the data'.format(self.sub_dir))
-
-    def bz(self):
-        return self.load_data(key='bug')
-
-    def bz_server_url(self):
-        return self.bz().data['server']
-
-    def bz_target_release(self):
```

Comment:
I double checked and these aren't used anywhere so they are good to be deleted

Reply:
Cool, going to merge this and set up a new PR to remove the obsolete methods